input/resources/CodeSystem-ehealth-program.json

```diff
@@ -0,0 +1,37 @@
+{
+  "resourceType": "CodeSystem",
+  "id": "ehealth-program",
+  "url": "http://ehealth.sundhed.dk/cs/ehealth-program",
+  "version": "2024-11-28T09:00:00",
+  "name": "eHealthProgram",
+  "title": "eHealth Program",
+  "status": "active",
+  "experimental": false,
+  "date": "2024-11-28T09:00:00+00:00",
+  "publisher": "ehealth.sundhed.dk",
+  "contact": [
+    {
+      "telecom": [
+        {
+          "system": "url",
+          "value": "http://ehealth.sundhed.dk/terminology"
+        }
+      ]
+    }
+  ],
+  "description": "A program/project of work for which this artifact is applicable.",
```

Comment:
description along the lines of VS is better:
Codes for programs or projects using the eHealth Infrastructure and wanting to mark various resources as intended for their non-exclusive use.